hbase-oss/src/main/java/org/apache/hadoop/hbase/oss/HBaseObjectStoreSemantics.java

```diff
@@ -58,6 +57,7 @@
 import org.apache.hadoop.hbase.oss.sync.TreeLockManager.Depth;
 import org.apache.hadoop.security.AccessControlException;
 import org.apache.hadoop.util.Progressable;
+import org.apache.hbase.thirdparty.com.google.common.annotations.VisibleForTesting;
```

Comment:
I like using hbase-thirdparty, but this means we need to have a declared dependency on it in the hbase-oss pom instead of relying on implicit inclusion via a transitive dependency.

If we're only using it for `VisibleForTesting` I'd also be cool with just removing that annotation and including a note in the javadoc.

Reply:
good point. in fact, the hbase1 profile has declares dependency on hbase-thirdparty, but not the hbase2 profile. I'll move the declaration out of the profile.

Reply:
Declared in provided scope.